iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/VoltageLevelXml.java

```diff
@@ -30,61 +33,76 @@ protected boolean hasSubElements(VoltageLevel vl) {
         return true;
     }
 
+    private TopologyLevel convertTopologyKindToTopologyLevel(TopologyKind topologyKind) {
+        switch (topologyKind) {
+            case NODE_BREAKER:
+                return TopologyLevel.NODE_BREAKER;
+            case BUS_BREAKER:
+                return TopologyLevel.BUS_BREAKER;
+            default:
+                return TopologyLevel.NODE_BREAKER;
+        }
+    }
```

Comment:
This method should be in the TopologyLevel enum:
- toTopologyKind()
- fromTopologyKind()